.github/workflows/update_docs.yml

```diff
@@ -0,0 +1,61 @@
+name: Update Docs
+
+on:
+  pull_request_target:
+    branches:
+      - main
+  pull_request:
```

Comment:
why are both needed? what are all the different scenarios when we want to trigger this workflow?

Reply:
one is for running on PR, and the other is for running when a pr merges to main

Reply:
```yml
on:
  push: [main]
  pull_request:
```

isn't that ☝️ what you want?

I can't really see why would
```yml
  pull_request_target:
    branches:
      - main
```
run when merging to `main` 🤔

Reply:
I need the PR labels so I can use on push. I will further test this on another branch, using a dummy branch as target to see if I understood the Github docs correctly.